src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -820,6 +823,51 @@ public void whenShouldNotSendContestedGeneralOrderEmailToRespondent_ThenTheEmail
         verify(notificationService, never()).sendContestedGeneralOrderEmailRespondent(any());
     }
 
+    @Test
+    public void whenToggleEnabledAndShouldSendEmailToRespSolicitor_thenSendsEmail() {
+        final ArgumentCaptor<CaseDetails> requestCaptor = ArgumentCaptor.forClass(CaseDetails.class);
+
+        when(featureToggleService.isRespondentSolicitorEmailNotificationEnabled()).thenReturn(true);
+        when(notificationService.shouldEmailRespondentSolicitor(any())).thenReturn(true);
+
+        notificationsController.sendAssignToJudgeConfirmationEmail(AUTH_TOKEN, buildCallbackRequest());
+
+        verify(notificationService).sendAssignToJudgeConfirmationEmailToRespondentSolicitor(any());
+
+        verify(notificationService).sendAssignToJudgeConfirmationEmailToRespondentSolicitor(requestCaptor.capture());
```

Comment:
you don't need to use an argument captor here (although I do love argument captors), it looks like you can just verify buildCallbackRequest().caseDetails is passed to this method call